lib/cob_index/macros/booster.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+require "traject"
+
+# This module has macros and methods related to boosting labels and fields.
+
+module CobIndex::Macros::Booster
+  # By convention we append "inverse_boost" or "boost" to the start of the labels as a signal to intent.
+  # Of course, how the label is actually being used depends on what is happening in the Solr configuration,
+  # or the query context.
+  def add_boost_labels
+    lambda do |rec, acc|
+      add_libraries_labels(rec, acc)
+      add_booksellers_labels(rec, acc)
+    end
+  end
+
+  private
+
+    def add_libraries_labels(rec, acc)
+      bad_libraries = [ "PRESSER", "CLAEDTECH" ]
```

Comment:
Please edit to "low_priority_libraries" here and elsewhere in this PR

Reply:
@ebtoner I decided to just remove that line.